environments/mcdTransformers.toml

```diff
@@ -1,1667 +1,1667 @@
+    migrations = "db/migrations"
+    contracts = ["MCD_VAT"]
+    rank = "0"
+  [exporter.vow]
+    path = "transformers/storage/vow/initializer"
+    type = "eth_storage"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    rank = "0"
+  [exporter.vow_fess]
+    path = "transformers/events/vow_fess/initializer"
+    type = "eth_event"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    contracts = ["MCD_VOW"]
+    rank = "0"
+  [exporter.vow_file_auction_address]
+    path = "transformers/events/vow_file/auction_address/initializer"
+    type = "eth_event"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    contracts = ["MCD_VOW"]
+    rank = "0"
+  [exporter.vow_file_auction_attributes]
+    path = "transformers/events/vow_file/auction_attributes/initializer"
+    type = "eth_event"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    contracts = ["MCD_VOW"]
+    rank = "0"
+  [exporter.vow_flog]
+    path = "transformers/events/vow_flog/initializer"
+    type = "eth_event"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    contracts = ["MCD_VOW"]
+    rank = "0"
+  [exporter.vow_heal]
+    path = "transformers/events/vow_heal/initializer"
+    type = "eth_event"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    contracts = ["MCD_VOW"]
+    rank = "0"
+  [exporter.yank]
+    path = "transformers/events/yank/initializer"
+    type = "eth_event"
+    repository = "github.com/makerdao/vdb-mcd-transformers"
+    migrations = "db/migrations"
+    contracts = [
+        "MCD_FLAP_1_0_0",
+        "MCD_FLAP_1_0_9",
+        "MCD_FLIP_BAT_A_1_0_0",
+        "MCD_FLIP_BAT_A_1_0_9",
+        "MCD_FLIP_BAT_A_1_1_0",
+        "MCD_FLIP_COMP_A_1_1_2",
+        "MCD_FLIP_ETH_A_1_0_0",
+        "MCD_FLIP_ETH_A_1_0_9",
+        "MCD_FLIP_ETH_A_1_1_0",
+        "MCD_FLIP_KNC_A_1_0_8",
+        "MCD_FLIP_KNC_A_1_0_9",
+        "MCD_FLIP_KNC_A_1_1_0",
+        "MCD_FLIP_LINK_A_1_1_2",
+        "MCD_FLIP_LRC_A_1_1_2",
+        "MCD_FLIP_MANA_A_1_0_9",
+        "MCD_FLIP_MANA_A_1_1_0",
+        "MCD_FLIP_PAXUSD_A_1_1_1",
+        "MCD_FLIP_SAI_1_0_0",
+        "MCD_FLIP_TUSD_A_1_0_7",
+        "MCD_FLIP_TUSD_A_1_0_9",
+        "MCD_FLIP_TUSD_A_1_1_0",
+        "MCD_FLIP_USDC_A_1_0_4",
+        "MCD_FLIP_USDC_A_1_0_9",
+        "MCD_FLIP_USDC_A_1_1_0",
+        "MCD_FLIP_USDC_B_1_0_7",
+        "MCD_FLIP_USDC_B_1_0_9",
+        "MCD_FLIP_USDC_B_1_1_0",
+        "MCD_FLIP_USDT_A_1_1_1",
+        "MCD_FLIP_WBTC_A_1_0_6",
+        "MCD_FLIP_WBTC_A_1_0_9",
+        "MCD_FLIP_WBTC_A_1_1_0",
+        "MCD_FLIP_ZRX_A_1_0_8",
+        "MCD_FLIP_ZRX_A_1_0_9",
+        "MCD_FLIP_ZRX_A_1_1_0",
+        "MCD_FLOP_1_0_1",
+        "MCD_FLOP_1_0_9",
+        "MCD_FLIP_ETH_B_1_1_3"
```

Comment:
I guess the same thing is true here, so maybe not worth it if it's going to create a ton of work to maintain